__frzr

```diff
@@ -789,6 +789,8 @@ prepare_efi_partition() {
 
 	local default_config_entry=""
 
+	local default_frzr_default_entry_filename="frzr.conf"
+
```

Comment:
Do we need to say default twice in the car?

Reply:
Renaming done